protocol-definitions/MC.yaml

```diff
@@ -955,3 +955,12 @@ methods:
           doc: |
             true if the session is found & closed,
             false otherwise.
+  - id: 37
+    name: demoteDataMember
+    since: 2.8
+    doc: |
+      Demotes the data member it's called on to a lite member.
+    request:
+      retryable: false
```

Comment:
why not retryable? is it not idempotent?

Reply:
Should we retry demoting the member after exception is thrown? I thought it won't add up any value.

Reply:
we should not retry it